e2e_tests/tests/fixtures/no_op/fail_on_first_validation.yaml

```diff
@@ -0,0 +1,26 @@
+description: noop_single
```

Comment:
Can we  reuse some other config file and use a temp config to set `fail_on_first_validation` in the code of the test?

Reply:
yeah, added `run_failure_test_with_temp_config` to allow this.